server/api/school/school.controller.js

```diff
@@ -113,6 +114,40 @@ exports.destroy = function(req, res) {
   });
 };
 
+function archiveStudents(id) {
+  return new Promise(function(resolve, reject) {
```

Comment:
What is the alternative if I want to return the value of students changed along with the status? When I tried to do it without a promise, the result object was not updated before the result was sent. If I tried to embed the return statement in this update statement, I got an undefined error.